dashboard/src/actions/authActions.js

```diff
@@ -6,16 +6,16 @@ import Cookies from "js-cookie";
 import { SUCCESS } from "assets/constants/overviewConstants";
 import { showToast } from "actions/toastActions";
 
+const maxWait = 50000; // Milliseconds
```

Comment:
I was thinking that this definition should go somewhere more like `src/assets/constants/authConstants.js`....  🤣